templates/auth/login.html

```diff
@@ -8,78 +8,45 @@
 {% endblock %}
 
 {% block main %}
+
+<!-- it's a preview, css just here to fix css in styleguide -->
 <style>
-form {
-  max-width: 400px;
-  margin: 0 auto;
-  padding: 20px;
-  border: 1px solid #ccc;
-  background-color: white;
-}
-label {
-  display: inline-block;
-  width: 100px;
-  margin-bottom: 20px;
-  font-weight: bold;
+main {
+  height: 100%;
 }
-input {
-  font: inherit;
-  padding: 0 5px;
-  border: 1px solid #ccc;
-  border-radius: 5px;
+#body {
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  align-items: center;
 }
-form button {
-  font: inherit;
-  font-weight: inherit;
-  background-color: #297ddb;
-  color: white;
-  border: 1px solid #6c6969;
-  border-radius: 7px;
-  padding: 5px 50px;
-  font-weight: bold;
-  margin-left: 100px;
+form.login[data-v-6e1bd991] {
+  max-width: 35rem;
 }
-
-@media screen and (max-width:400px) {
-  label,
-  input,
-  form button {
-    width: 100%;
-  }
-
-  label {
-    margin-bottom: 0.5rem;
-  }
-
-  input {
-    margin-bottom: 2rem;
-  }
-  form button {
-    margin: 0;
-  }
+form > div > label[data-v-6e1bd991] {
+  width: 9rem
 }
 </style>
 
-{% block login %}
-<form action={{ url_for('login') }} method=post>
-  {% if errors %}
-    <ul>
-    {% for error in errors %}
-      <li class=error>{{ error }}</li>
-    {% endfor %}
-    </ul>
-  {% endif %}
-  <h1>Log into my account</h1>
-  <div>
-    <label for=email>Email</label>
-    <input type=email id=email name=email required>
-  </div>
-  <div>
-    <label for=password>Password</label>
-    <input type=password id=password name=password required>
-  </div>
-  <button type=submit>Log in</button>
-  <p class=alternative><a href={{ url_for('signup') }}>I don't have an account yet!</a></p>
-</form>
+<section id=body>
+  {% block login %}
+  <x-form class=login action={{ url_for('login') }} method=post>
+    <div class=form-row>
+      <label for=email>Email</label>
+      <input type=email name=email id=email>
+    </div>
+    <div class=form-row>
```

Comment:
`class=form-row` is actually not required: https://github.com/neomad-team/styleguide/blob/vuejs-neomad/src/components/x-form.vue#L19

We should adapt the styleguide demo also.